src/tools/haxelib/Main.hx

```diff
@@ -252,7 +254,7 @@ class Main {
 			if (cats[i] == null) cats[i] = [c];
 			else cats[i].push(c);
 		}
-		
+
```

Comment:
Any reason for this? We eventually want to make haxelib usable as a library (see #186), so making this global seems a step in the wrong direction.